api/auth/auth-middleware.js

```diff
@@ -16,11 +17,27 @@ const restricted = (req, res, next) => {
 
     Put the decoded token in the req object, to make life easier for middlewares downstream!
   */
-  next();
-};
+const restricted = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
```

Comment:
pai il citesti din headers sau din cookie totusi?

Reply:
cum e mai bine? in conditie scrie ca trebuie din headers de citit.

Reply:
Deci ca in conditie e okay